src/neuroconv/datainterfaces/ecephys/axona/axona_utils.py

```diff
@@ -365,14 +367,18 @@ def get_position_object(file_path: FilePath) -> Position:
     be preferred to read position data from the `.bin` file to ensure
     samples are locked to ecephys time courses.
 
-    Parameters:
+    Parameters
     ----------
-    file_path (Path or Str):
+    file_path : Path or str
```

Comment:
I did not use Union here as it did in some other cases.

Reply:
yeah, curious...